<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1021</width>
    <height>873</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>850</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MissionConverter (Pixhawk 4)</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout" columnstretch="0,1">
    <item row="2" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1" rowspan="4">
     <layout class="QVBoxLayout" name="mapVerticalLayout" stretch="0,1">
      <item>
       <layout class="QHBoxLayout" name="mapHorizontalLayout">
        <item>
         <widget class="QLineEdit" name="pointLatLng">
          <property name="toolTip">
           <string>latitude, longitude</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
          </property>
          <property name="placeholderText">
           <string>latitude, longitude</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWebEngineView" name="mapView"/>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="coordinates">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Coordinates</string>
        </property>
        <layout class="QGridLayout" name="coordinatesGridLayout">
         <item row="0" column="1">
          <widget class="QLabel" name="labelLatitude">
           <property name="text">
            <string>Latitude</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="labelLongitude">
           <property name="text">
            <string>Longitude</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="labelNumber">
           <property name="text">
            <string>#</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="labelAltitude">
           <property name="text">
            <string>Altitude, abs (m)</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="parameters">
        <property name="title">
         <string>Parameters</string>
        </property>
        <layout class="QGridLayout" name="parametersGridLayout">
         <item row="5" column="0">
          <widget class="QLabel" name="labelWPRadius">
           <property name="text">
            <string>wpRadius (m)</string>
           </property>
          </widget>
         </item>
         <item row="9" column="0" colspan="2">
          <widget class="QCheckBox" name="removeParametersFileAfterRead">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Remove parameters file after read</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="labelStabilizeMotorPower">
           <property name="text">
            <string>Stabilize motor power (%)</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QSpinBox" name="gpsMaxWaitTime">
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
           </property>
           <property name="minimum">
            <number>-1000</number>
           </property>
           <property name="maximum">
            <number>3000</number>
           </property>
           <property name="value">
            <number>20</number>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="labelFlightHeightOffset">
           <property name="text">
            <string>flightHeight offset (m)</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QDoubleSpinBox" name="flightHeightOffset">
           <property name="locale">
            <locale language="English" country="UnitedStates"/>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
           </property>
           <property name="minimum">
            <double>-1000.000000000000000</double>
           </property>
           <property name="maximum">
            <double>3000.000000000000000</double>
           </property>
           <property name="value">
            <double>5.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="startDelayInSec">
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
           </property>
           <property name="maximum">
            <number>500</number>
           </property>
           <property name="value">
            <number>300</number>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="labelStabilizeFlightTime">
           <property name="text">
            <string>Stabilize flight time (seconds)</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="labelFlightHeight">
           <property name="text">
            <string>Flight height (m)</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="labelStartDelay">
           <property name="text">
            <string>Start delay (seconds)</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QSpinBox" name="stabilizeFlightTime">
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
           </property>
           <property name="minimum">
            <number>-1000</number>
           </property>
           <property name="maximum">
            <number>3000</number>
           </property>
           <property name="value">
            <number>60</number>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="fbwaMotorPower">
           <property name="locale">
            <locale language="English" country="UnitedStates"/>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
           </property>
           <property name="minimum">
            <double>0.000000000000000</double>
           </property>
           <property name="maximum">
            <double>90.000000000000000</double>
           </property>
           <property name="value">
            <double>90.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelFBWAMotorPower">
           <property name="text">
            <string>FBWA motor power (%)</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0" colspan="2">
          <widget class="QCheckBox" name="enableLogging">
           <property name="text">
            <string>Enable Logging</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QDoubleSpinBox" name="flightHeight">
           <property name="locale">
            <locale language="English" country="UnitedStates"/>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
           </property>
           <property name="minimum">
            <double>-1000.000000000000000</double>
           </property>
           <property name="maximum">
            <double>3000.000000000000000</double>
           </property>
           <property name="value">
            <double>150.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QDoubleSpinBox" name="stabilizeMotorPower">
           <property name="locale">
            <locale language="English" country="UnitedStates"/>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
           </property>
           <property name="minimum">
            <double>0.000000000000000</double>
           </property>
           <property name="maximum">
            <double>100.000000000000000</double>
           </property>
           <property name="value">
            <double>60.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QSpinBox" name="wpRadius">
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
           </property>
           <property name="minimum">
            <number>-1000</number>
           </property>
           <property name="maximum">
            <number>3000</number>
           </property>
           <property name="value">
            <number>5</number>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="labelGPSMaxWaitTime">
           <property name="text">
            <string>GPS max wait time (seconds)</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="generateParameters">
      <property name="text">
       <string>Generate parameters file (Ctrl-G)</string>
      </property>
      <property name="shortcut">
       <string>Ctrl+G</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header location="global">QtWebEngineWidgets/QWebEngineView</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>startDelayInSec</tabstop>
  <tabstop>fbwaMotorPower</tabstop>
  <tabstop>stabilizeMotorPower</tabstop>
  <tabstop>flightHeight</tabstop>
  <tabstop>flightHeightOffset</tabstop>
  <tabstop>wpRadius</tabstop>
  <tabstop>stabilizeFlightTime</tabstop>
  <tabstop>gpsMaxWaitTime</tabstop>
  <tabstop>enableLogging</tabstop>
  <tabstop>removeParametersFileAfterRead</tabstop>
  <tabstop>generateParameters</tabstop>
  <tabstop>pointLatLng</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
